packages/inter-protocol/scripts/add-initial-liquidity.js

```diff
@@ -0,0 +1,186 @@
+// @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { makeNotifierFromAsyncIterable } from '@agoric/notifier';
+import { E } from '@endo/far';
+
+const { entries } = Object;
+
+const dec6 = 10n ** 6n;
+const dec4 = 10n ** 4n;
+
+const config = {
+  Central: { IST: 10_000n * dec6 },
+  Secondary: { IbcATOM: 500n * dec4 }, // 20 IST per ATOM
+};
+
+// /** @type {<K extends string,V>(es: [K,V][]) => Record<K,V>} */
+// const recordFromEntries = Object.fromEntries;
+
+const getOfferResult = async (instance, wallet, walletAdmin) => {
+  const offerIt = E(walletAdmin).getOffersNotifier();
+  let result;
+  console.log('awaiting offers...');
+  for await (const offers of makeNotifierFromAsyncIterable(offerIt)) {
+    console.log('offers iterator:', offers);
```

Comment:
this is a next of the iterator, not the iterator itself

Reply:
fixed